eng/Versioning.props

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+  <PropertyGroup>
+    <_BuildNumber>$(OfficialBuildId)</_BuildNumber>
+    <_BuildNumber Condition="'$(OfficialBuildId)' == ''">$([System.DateTime]::Now.ToString(yyyyMMdd)).1</_BuildNumber>
+
+    <_PreReleaseLabel>$(PreReleaseVersionLabel)</_PreReleaseLabel>
+    <_PreReleaseLabel Condition="'$(_PreReleaseLabel)' == ''">dev</_PreReleaseLabel>
```

Comment:
Do we want to allow publishing non-nightly packages with dev label? Like 1.0-dev? Might be better to error out in the case where `DotNetFinalVersionKind` is `prerelease` but `PreReleaseVersionLabel` is not overridden.

Reply:
Probably not worth having an error as there may be end up some cases where we actually want to do that.